template/de-DE/schannel.adml

```diff
@@ -82,6 +82,14 @@ Damit die Änderungen übernommen werden, muss der Computer neugestartet werden.
 Damit die Änderungen übernommen werden, muss der Computer neugestartet werden.
             </string>
 
+            <!-- TLSv1.3 -->
+            <string id="TLSv13">TLS 1.3 [EXPERIMENTAL]</string>
+            <string id="TLSv13_Help">Aktiviert oder deaktiviert TLS 1.3.  TLS 1.3 hat keine bekannten Sicherheitsprobleme.
+
+This setting is only compatible on Windows 10 1903 and above and does not require a reboot to take effect.

```

Comment:
My German is too rusty to come up with the translation on my own, but as far as I can tell (and remember), Google Translate did a pretty good job of it here:

```suggestion
Diese Einstellung ist nur unter Windows 10 1903 und höher kompatibel und erfordert keinen Neustart.
```

If a native speaker comes along (@teddue, maybe? 😄 He wrote the original translation!) and decides that this is a terrible translation, I have no qualms about changing it at that time.